application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -25,7 +25,7 @@
  */
 public final class TagCommand extends SlashCommandAdapter {
     private final TagSystem tagSystem;
-
+    private static final int MAX_OPTIONS = 5;
```

Comment:
`MAX_OPTIONS` feels a bit confusing, naming-wise. i think it would be clearer if its `MAX_SUGGESTIONS` or `MAX_MATCHES`.